packages/task/src/browser/quick-open-task.ts

```diff
@@ -322,6 +307,31 @@ export class QuickOpenTask implements QuickOpenModel, QuickOpenHandler {
         });
     }
 
+    private addTaskItemsByTaskScope(groupedTasks: Map<string, TaskConfiguration[]>, token: number, isMulti: boolean, isFirstGroup: boolean, taskScopeString: string): boolean {
+        let showBorder = isFirstGroup ? false : true;
```

Comment:
minor: no need for a ternary operator

```suggestion
        let showBorder = !isFirstGroup
```

Reply:
The new refactored code in the latest commit does no longer require the showBorder variable. Thaks !